src/data.rs

```diff
@@ -20,16 +21,19 @@ pub enum Struct<T> {
 #[derive(Debug, Clone)]
 pub struct UnitStruct {
```

Comment:
Please add an `attrs()` method for UnitStruct as well, otherwise the attrs in here are not accessible in the public API.

Consider also adding `attrs()` on Data and Struct.